packages/vite/src/node/optimizer/index.ts

```diff
@@ -47,9 +47,29 @@ export interface DepOptimizationOptions {
    */
   exclude?: string[]
   /**
-   * The bundler sometimes needs to rename symbols to avoid collisions.
-   * Set this to `true` to keep the `name` property on functions and classes.
-   * https://esbuild.github.io/api/#keep-names
+   * Options to pass to esbuild during the dep scanning and optimization
+   *
+   * - `plugins` are merged with Vite's dep plugin
+   * - `keepNames` takes precedence over the deprecated `optimizeDeps.keepNames`
+   * - `external` is omitted, use Vite's optimizeDeps.exclude option
+   *
+   * https://esbuild.github.io/api
```

Comment:
This make the link clickable in VSCode

```suggestion
   * [esbuild.github.io/api](https://esbuild.github.io/api)
```

Reply:
Odd, it's clickable for me in any case

Reply:
Checked it and you are right. Strange, thought it doesn't work :shrug: